src/org/traccar/geocoder/JsonGeocoder.java

```diff
@@ -113,4 +114,8 @@ public void failed(Throwable throwable) {
 
     public abstract Address parseAddress(JsonObject json);
 
+    protected String getError(JsonObject json) {
+        return "";
```

Comment:
I don't like this default. It should return `null`. Also, I would rename it to `parseError` for consistency.

Reply:
renamed. Not sure about the null, 'cause it'll generate logs like:
"Empty address. null"